tests/test_payments.py

```diff
@@ -196,56 +188,66 @@ def test_list_payments_with_ids_and_other_params(self):
-        mock_ps_add.assert_called_with(mock.ANY, DEBIT)
+        mock_ps_add.assert_called_with(DEBIT)
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
         self.assertEqual(json.loads(resp.data), {'created' : DC_RETURN})
 
-    @mock.patch.object(PaymentService, 'add_payment', side_effect=DataValidationError, autospec=True)
-    def test_crud_create_bad_data(self, mock_ps_add):
+    @mock.patch.object(PaymentService, 'add_payment', side_effect=DataValidationError)
+    def test_crud_create_bad_data_single_quotes(self, mock_ps_add):
         data = json.dumps(BAD_DATA)
-        try:
-            resp = self.app.post('/payments', data=data, content_type='application/json')
-        except DataValidationError as e:
-            self.assertTrue('missing nickname' in e.message)
-            mock_ps_add.assert_called_with(mock.ANY, BAD_DATA)
-            self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST)
-            self.assertTrue('missing nickname' in json.loads(resp.data))
-
-    @mock.patch.object(PaymentService, 'add_payment', side_effect=DataValidationError, autospec=True)
+        
+        resp = self.app.post('/payments', data=data, content_type='application/json')
+        mock_ps_add.assert_called_with(BAD_DATA)
+        self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertTrue("error" in resp.data)
+
+    @mock.patch.object(PaymentService, 'add_payment', side_effect=DataValidationError)
+    def test_crud_create_bad_data_double_quotes(self, mock_ps_add):
+        data = json.dumps(BAD_DATA2)
+        
+        resp = self.app.post('/payments', data=data, content_type='application/json')
+        mock_ps_add.assert_called_with(BAD_DATA2)
+        self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertTrue("error" in resp.data)
+
+    @mock.patch.object(PaymentService, 'add_payment', side_effect=DataValidationError)
     def test_crud_create_garbage(self, mock_ps_add):
         garbage = 'a@$*&@#sdassdc3r 3284723X43&^@!#@*#'
-        data = json.dumps(garbage)
-        try:
-            resp = self.app.post('/payments', data=data, content_type='application/json')
-        except DataValidationError as e:
-            self.assertTrue('bad or no data' in e.message)
-            mock_ps_add.assert_called_with(mock.ANY, None)
-            self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+        
+        resp = self.app.post('/payments', data=garbage, content_type='application/json')
+        mock_ps_add.assert_called_with(None)
+        self.assertEqual(resp.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertTrue("error" in resp.data)
+
+    def test_index(self):
+        resp = self.app.get('/')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        self.assertTrue('Welcome to payments!' in resp.data)
```

Comment:
I'm assuming this is basically the only *actual* change introduced by the branch? :D